exampleCourse/questions/demoAutograderNumpy/tests/setup_code.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import numpy.linalg as la
+
+# set up parameters
```

Comment:
Does the test class automatically handle setting equivalent seeds between student code and the solution code?

Reply:
No, that needs to be done manually in the setup code.  I can add an example question for this.

Reply:
Actually, I wonder if it would be worthwhile to just always seed user code and answer code with the same seed.  I don't think there are any downsides to doing this?

Reply:
I'm in favor of it... The only issue I can think of is each library defines their own set seed function variant. e.g.

```py
random.seed( 30 ) #random
numpy.random.seed(30) #numpy 
tf.random.set_seed  (30) #tensorflow
torch.manual_seed(30) #pytorch
```

Reply:
bleh, that's annoying and I can see it giving problems if some random esoteric library is used.  But I guess in those cases the seed can just be set manually?